js/components/frame.js

```diff
@@ -54,6 +54,10 @@ class Frame extends ImmutableComponent {
 
   componentDidMount () {
     this.updateWebview()
+    // forward postMessage events from webview to webContents
+    window.addEventListener('message', function (event) {
```

Comment:
I think I should add an origin check here. The page itself will have no access to the embedder and can't send it any messages, but I think this could end up firing multiple messages for the same event because "window" is the same for all frames. Going to create a quick patch for it right now.